owncloudApp/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -115,6 +115,8 @@ public class OCFileListFragment extends ExtendedListFragment implements
 
     private static final String GRID_IS_PREFERED_PREFERENCE = "gridIsPrefered";
 
+    private static final String ACTION_BUTTON = "ACTION_BUTTON";
+
```

Comment:
Since this is an EXTRA from the FolderPicker, and it already has the key there, we should not create the same key in any other file. Don't you think?

Reply:
Done